contracts-periphery/src/UniswapWithdrawHook.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+pragma abicoder v2;
+
+import "openzeppelin-contracts/contracts/access/Ownable.sol";
+import "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import "src/interfaces/ISwapRouter.sol";
+
+contract UniswapWithdrawHook is Ownable {
+  using SafeERC20 for IERC20;
+
+  ISwapRouter internal immutable swapRouter;
+  address internal WETH9;
+
+  uint256 public feeBips;
+  address public feeCollector;
+  address payable public feeReceiver;
+
+  constructor(ISwapRouter _swapRouter, address _WETH9, uint256 _feeBips, address payable _feeReceiver) {
+  swapRouter = _swapRouter;
```

Comment:
Indentation in the constructor is off

Reply:
Fixed in the same commit as above https://github.com/ScopeLift/umbra-protocol/pull/341/commits/75faac7fcc1bbfed25b90ee04633145ddcd72693